tests/jenkins/TestUploadMinSnapshotsToS3.groovy

```diff
@@ -8,31 +8,21 @@
 
 package jenkins.tests
 
+import com.kenai.jffi.Closure
+import jenkins.tests.BuildPipelineTest
 import org.junit.*
-import java.util.*
 
 class TestUploadMinSnapshotsToS3 extends BuildPipelineTest {
 
     @Before
     void setUp() {
+        List <Closure> fileActions = ['createSha512Checksums']
+        this.registerLibTester(new UploadMinSnapshotsToS3LibTester( fileActions, 'tests/jenkins/data/opensearch-1.3.0.yml' ))
```

Comment:
Since we are not able to import the lib from vars, I treat this one as a string for comparison @dblock

Reply:
If there is a way to import it into unit test, please let me know about it.

Reply:
I don't understand the problem, what would you like to write here instead of what's in the code?

Reply:
The original code is:
```
         List <Closure> fileActions = [createSha512Checksums()] 
```
which does not run because it cannot figure out what is `createSha512Checksums()` as I cant seems to find a way to import it from `/vars/createSha512Checksums.groovy`.

Reply:
I see, you could pass another dummy function in here instead of `createSha512Checksums`?

Reply:
I can, but that kinda defeat the purpose of testing this.
Will try in a different test tho.
Thanks.

Reply:
@abhinavGupta16 thoughts?

Reply:
I don't think a dummy function would do the job either since the passed function in the job is a closure call. The `class.name` fetches `createSha512Checksums$closurecall_1` and not `createSha512Checksums`

Reply:
I tried declaring a closure but it doesn't get a name. I would just leave this as is.